src/components/Button/button.scss

```diff
@@ -0,0 +1,48 @@
+@import "../../styles/vars";
+@import "../../styles/extends";
```

Comment:
no need to import these files each time. you can do it in the `main.scss` once

Reply:
I imported those files to index.scss, but still it doesn't work, only importing to separate scss files makes variables visible in the file.